### ISSUE TITLE
Not able to generate docs

### ISSUE THREAD

**Issue by CuriousLearner** [davidevernizzi/docman#6]
I'm not sure why this is happening, but the first two methods document well, as soon as I add this third method ( and other methods ) the docs are not generated.

```
{
    "id": "3512fffd-3b4f-5c5f-bb23-14736ee69074",
    "name": "Shopster",
    "description": "",
    "order": [
        "09c67afd-0966-0faa-39a2-13ebe0d5df00",
        "0773b887-360e-40c8-94cf-9cc41d28a251",
        "121a2941-84e9-dd77-d7b2-9769007559ee"
    ],
    "folders": [],
    "timestamp": 1457791398412,
    "owner": 0,
    "remoteLink": "",
    "public": false,
    "requests": [
        {
            "id": "0773b887-360e-40c8-94cf-9cc41d28a251",
            "headers": "Content-Type: application/json\n",
            "url": "http://localhost:8000/api/recharge/",
            "preRequestScript": null,
            "pathVariables": {},
            "method": "POST",
            "data": [],
            "dataMode": "raw",
            "version": 2,
            "tests": null,
            "currentHelper": "normal",
            "helperAttributes": {},
            "time": 1459976600672,
            "name": "Recharge wallet",
            "description": "",
            "collectionId": "3512fffd-3b4f-5c5f-bb23-14736ee69074",
            "responses": [],
            "rawModeData": "{\n    \"ordered_by\": 1,\n    \"coupon_code\": \"hello\"\n}"
        },
        {
            "id": "09c67afd-0966-0faa-39a2-13ebe0d5df00",
            "headers": "Content-Type: application/json\n",
            "url": "http://localhost:8000/api/products/",
            "preRequestScript": null,
            "pathVariables": {},
            "method": "POST",
            "data": [],
            "dataMode": "raw",
            "version": 2,
            "tests": null,
            "currentHelper": "normal",
            "helperAttributes": {},
            "time": 1459976647632,
            "name": "Add Product Details",
            "description": "",
            "collectionId": "3512fffd-3b4f-5c5f-bb23-14736ee69074",
            "responses": [],
            "rawModeData": "{\n    \"price\": 12,\n    \"name\": \"maggi\",\n    \"quantity\": 123,\n    \"category_id\": null\n}"
        },
        {
            "id": "121a2941-84e9-dd77-d7b2-9769007559ee",
            "headers": "",
            "url": "http://localhost:8000/api/user/1",
            "preRequestScript": null,
            "pathVariables": {},
            "method": "GET",
            "data": null,
            "dataMode": "params",
            "version": 2,
            "tests": null,
            "currentHelper": "normal",
            "helperAttributes": {},
            "time": 1459977048678,
            "name": "Get User by ID",
            "description": "",
            "collectionId": "3512fffd-3b4f-5c5f-bb23-14736ee69074",
            "responses": []
        }
    ]
}
```

**Comment by rowasc** [comment]
Hi, just saw this and thought I'd give it a try . 
Apparently the issue is with GET requests, because the request.data generated on those requests is now NULL, where it used to be an empty array.  
I will push a fix in a few minutes and open the PR
Romina

**Comment by CuriousLearner** [comment]
@rowasc I think, you're right. Changing from null to empty array generates the doc.

We shall wait, to get your code pushed in :)

Thanks :+1:

**Comment by rowasc** [comment]
Fixed in #8 .
Thanks
Romina
